configs/components/oifs/oifs.yaml

```diff
@@ -425,6 +425,8 @@ file_movements:
             all_directions: copy
         waminfo_in:
             all_directions: copy
+        outdata: # TODO: ask GEOMAR if they are fine with this, otherwise we have to put it into awicm3.yaml
```

Comment:
have you asked?

Reply:
@joakimkjellsson are you okay with this:
https://github.com/esm-tools/esm_tools/blob/cb2a6341a9a0ef33883fe08529b0d58c72b95a07/configs/components/oifs/oifs.yaml#L428-L429

If you are not okay with all output files from OpenIFS (both in OpenIFS-standalone and FOCI) being moved instead of copied, we can add this only to the awicm3.yaml.